src/univariate/continuous/weibull.jl

```diff
@@ -139,3 +139,41 @@ end
 #### Sampling
 
 rand(rng::AbstractRNG, d::Weibull) = xv(d, randexp(rng))
+
+#### Fit model
+
+"""
+    fit_mle(::Type{<:Weibull}, x::AbstractArray{T}; 
+    α0::Float64 = 1.0, maxiter::Int = 1000, tol::Float64 = 1e-16)
+
+Maximum Likelihood Estimate of `Weibull` Distribution via Newton's Method
+
+"""
+function fit_mle(::Type{<:Weibull}, x::AbstractArray{T};
+    α0::Float64 = 1.0, maxiter::Int = 1000, tol::Float64 = 1e-16) where T <: Real
```

Comment:
The type parameter is not needed it seems and `Float64` might be a bit too restrictive. Additionally, probably it would be good to not use Unicode characters in the official API but at least provide an ASCII alternative.
```suggestion
    fit_mle(::Type{<:Weibull}, x::AbstractArray{<:Real}; 
    alpha0::Real = 1, maxiter::Int = 1000, tol::Real = 1e-16)

Compute the maximum likelihood estimate of the [`Weibull`](@ref) distribution with Newton's method.
"""
function fit_mle(::Type{<:Weibull}, x::AbstractArray{<:Real};
    alpha0::Real = 1, maxiter::Int = 1000, tol::Real = 1e-16)
```